src/AppBundle/Command/RemoveImageCommand.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace AppBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\PropertyAccess\PropertyAccessor;
+
+class RemoveImageCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('app:remove-image')
+            ->setDescription('Remove image that not relation to media')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $mm = $this
+            ->getContainer()
+            ->get('sonata.media.manager.media')
+        ;
+
+        $em = $this
+            ->getContainer()
+            ->get('doctrine')
+            ->getManager()
+        ;
+
+        $medias = $mm->findAll();
+
+        foreach ($medias as $media) {
+            $context = $media->getContext() == "slider" ? "performance" : $media->getContext();
+            $entity = "AppBundle\\Entity\\".ucfirst($context);
+
+            switch ($media->getContext()) {
+                case 'employee':
```

Comment:
So. What will happen if I'll add/edit/delete context? I'll need to edit command, right? 
Move this logic into separate method, and rewrite it all.